Pod/Classes/KolodaView/KolodaView.swift

```diff
@@ -85,6 +85,20 @@ public class KolodaView: UIView, DraggableCardDelegate {
     public var alphaValueSemiTransparent: CGFloat = defaultAlphaValueSemiTransparent
     
     //MARK: Lifecycle
+    required public init(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+        configure()
+    }
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        configure()
+    }
+    
+    deinit {
+        NSNotificationCenter.defaultCenter().removeObserver(self)
```

Comment:
How do feel to replace this line to a separate method like 'unsubscribeFromNotifications'